vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -1055,8 +1055,11 @@ class RoomDetailFragment @Inject constructor(
         val data = event.buildImageContentRendererData(dimensionConverter.dpToPx(66))
         val isImageVisible = if (data != null) {
             imageContentRenderer.render(data, ImageContentRenderer.Mode.THUMBNAIL, views.composerLayout.views.composerRelatedMessageImage)
+            views.composerLayout.views.composerRelatedMessageImage.isVisible = true
             true
         } else {
+            imageContentRenderer.clear(views.composerLayout.views.composerRelatedMessageImage)
+            views.composerLayout.views.composerRelatedMessageImage.isVisible = false
```

Comment:
Maybe simplify the code duplication by moving this line outside of the if and else block and just write:
```kotlin
views.composerLayout.views.composerRelatedMessageImage.isVisible = isImageVisible
```

Reply:
Yes that does make a lot of sense, I will update the code, thank you for the review!